cpp/src/arrow/c/bridge_test.cc

```diff
@@ -353,6 +353,8 @@ TEST_F(TestSchemaExport, Primitive) {
   TestPrimitive(large_binary(), "Z");
   TestPrimitive(utf8(), "u");
   TestPrimitive(large_utf8(), "U");
+  TestPrimitive(binary_view(), "vz");
+  TestPrimitive(utf8_view(), "vu");
 
```

Comment:
Should I add a test exercising many/no buffers?

Reply:
That would be nicer IMHO.